src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +753,70 @@ const UsersConfigurationRow = ({
     );
 };
 
+const SshKeysRow = ({
+    id, item, onChange, idx, removeitem,
+}) => {
+    const [isLoading, setIsLoading] = useState(false);
+
+    const handleClear = (_event) => {
+        onChange(idx, "file", "");
+        onChange(idx, "value", "");
+    };
+
+    return (
+        <Grid id={id}>
+            <GridItem span={11}>
+                <FileUpload id={"create-vm-dialog-ssh-key-" + id}
+                    type="text"
+                    value={item.value}
+                    filename={item.filename}
+                    filenamePlaceholder="Drag and drop a file or upload one"
+                    onFileInputChange={(_event, file) => onChange(idx, "file", file)}
+                    onDataChange={(_event, value) => onChange(idx, "value", value)}
+                    onTextChange={(_event, value) => onChange(idx, "value", value)}
+                    onReadStarted={() => setIsLoading(true)}
+                    onReadFinished={() => setIsLoading(false)}
+                    onClearClick={_event => handleClear(_event)}
+                    isLoading={isLoading}
+                    allowEditingUploadedText
+                    browseButtonText="Upload" />
+            </GridItem>
+            <GridItem span={1} className="pf-m-1-col-on-md remove-button-group">
+                <Button variant='secondary'
+                    className="btn-close"
+                    id={id + "-btn-close"}
+                    isSmall
+                    aria-label={_("Remove item")}
+                    icon={<MinusIcon />}
+                    onClick={() => removeitem(idx)} />
+            </GridItem>
+        </Grid>
```

Comment:
These 28 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20230718-074723-da6d3da1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)